### PR TITLE
AB#38578 string verification

### PR DESCRIPTION
**Pull request by gracet9182**
## Overview

* Did a string review with Laura Graham
* These are the changes she recommended

### CODE REVIEW

File: src/messages/GE.json [hunk 1]
```diff
@@ -356,13 +356,12 @@
...
-  "Sample Explorer Teams app": "Sample Explorer Teams app",
-  "Application Permissions": "Application Permissions",
+  "As user": "(signed in as user)",
+  "Use Graph Explorer as a sample Teams application": "Use Graph Explorer as a sample Teams application",
+  "Use Graph Explorer as a signed in user": "Use Graph Explorer as a signed in user",
+  "Resource Specific Consent popup": "To use resource-specific consent permissions, download​",
+  "Sample Teams app for Graph Explorer": "Sample Teams app for Graph Explorer",
+  "Install sample app": "Install sample app",
   "Learn more about": "Learn more about",
   "resource specific consent": "resource specific consent",
```
Comment (acchiang) [inline src/messages/GE.json]: does this need a hyphen between resource and specific as well?

Comment (gracet9182) [reply src/messages/GE.json]: Ah, wasn't thinking about internal variables! ad3c65b773d07bd438c8de531f8a0e8161f03bda

File: src/messages/GE.json [hunk 2]
```diff
@@ -359,11 +359,10 @@
...
   "Use Graph Explorer as a sample Teams application": "Use Graph Explorer as a sample Teams application",
   "Use Graph Explorer as a signed in user": "Use Graph Explorer as a signed in user",
-  "Resource Specific Consent popup": "To use resource-specific consent permissions, download​",
+  "Resource-specific Consent popup": "To use resource-specific consent permissions, download​",
   "Sample Teams app for Graph Explorer": "Sample Teams app for Graph Explorer",
   "Install sample app": "Install sample app",
   "Learn more about": "Learn more about",
-  "resource specific consent": "resource specific consent",
-  "Graph Explorer Sample App": "Graph Explorer Sample App",
+  "resource-specific consent": "resource specific consent",
```
Comment (acchiang) [inline src/messages/GE.json]: you forgot the second hyphen in the right hand side :)

File: src/messages/GE.json (hunk 1 above)
Comment (acchiang) [reply src/messages/GE.json]: np! if it's not outside your scope maybe you can do a repo-wide search for "resource consent" to see how widespread this missing hyphenation issue is

File: src/messages/GE.json (hunk 2 above)
Comment (gracet9182) [reply src/messages/GE.json]: d91f60424212896bb86acf9751c22a1698a89cbb

**Review by acchiang — APPROVED** [review]
lgtm ty!

**Review by lhao03 — APPROVED** [review]
noice

**Pull request merged**